src/libs/brands.ts

```diff
@@ -290,6 +328,61 @@ export const brands: Record<string, BrandConfiguration> = {
       colorPalette: { ...baseColors, primary: '#521b93', secondary: '#011c48', accent: '#521b93' },
     },
   },
+  publicHealth: {
+    name: 'Terra for Public Health Genomics',
+    queryName: 'publicHealth',
+    welcomeHeader: 'Terra for Public Health Genomics',
+    description:
+      'Terra is a secure cloud-based data platform enabling pathogen genomic data analysis in public health, clinical, and academic settings.',
+    hostName: 'publichealth.terra.bio',
+    docLinks: [
+      {
+        link: 'https://dockstore.org/organizations/Theiagen',
+        text: 'Theiagen Workflow Repository',
+      },
+      {
+        link: 'https://docs.google.com/document/d/e/2PACX-1vTPQC-bYMRxXQ4Gz9ESH_Eo-E6UXD2qOL7_3iMxJaQF3pOyW3tUyu7G9Nvk-JTf8xDkOyhftvd9L-sa/pub',
+        text: 'Terra Cloud Costs FAQs',
+      },
+      {
+        link: 'mailto:publichealthgenomics@broadinstitute.org',
+        text: 'Contact us',
+      },
+      {
+        link: 'https://support.terra.bio/hc/en-us',
+        text: 'Terra Support',
+      },
+    ],
+    logos: {
+      color: publicHealthLogo,
+      white: publicHealthLogoWhite,
+    },
+    landingPageBackground: publicHealthBackground,
+    landingPageCards: [
+      {
+        link: 'workspaces',
+        title: 'My Workspaces',
+        body: 'Workspaces connect your data to popular analysis tools powered by the cloud. Use Workspaces to share data, code, and results easily and securely.',
+      },
+      {
+        link: 'workspaces',
+        title: 'Examples',
+        body: 'Browse example pathogen genomic data and analysis tools together.',
+        linkPathParams: {},
+        linkQueryParams: { tab: 'public', 'tagsFilter[]': 'pathogen genomics' },
```

Comment:
Since all 3 of these options are related to the link, can we namespace them under one object like:

```suggestion
        title: 'Examples',
        body: 'Browse example pathogen genomic data and analysis tools together.',
        link: {
          routeName: 'workspaces',
          params: {},
          query: { tab: 'public', 'tagsFilter[]': 'pathogen genomics' },
        }
```